ACMWebSite/settings.py

```diff
@@ -124,7 +124,13 @@
 EMAIL_HOST_USER = environ.get('EMAIL_HOST_USER', 'email')
 EMAIL_HOST_PASSWORD = environ.get('EMAIL_HOST_PASSWORD', 'password')
 
+#for developement
 CORS_ORIGIN_ALLOW_ALL = True
-CORS_ORIGIN_WHITELIST = (
-	environ.get('REACT_JS_SERVER_IP', '200.3.154.201')
+
+#for production (only the front end server should be here)
+'''CORS_ORIGIN_WHITELIST = (

```

Comment:
This commented code has a syntax error, parentheses '()' should be changed 
with square brackets '[]' reffer to the official documentation of CORS headers https://pypi.org/project/django-cors-headers/ for more information.